src/Deck.js

```diff
@@ -0,0 +1,11 @@
+class Deck {
```

Comment:
I realize that the way I'm committing, it is undeterminable what is written first. I have written the tests first for everything. At this point I'm not writing all the tests and then the entire class but test to class to test to class.